core-bundle/contao/library/Contao/Image.php

```diff
@@ -148,7 +148,7 @@ public static function getHtml($src, $alt='', $attributes='')
 
 			if (file_exists(Path::join($projectDir, $darkSrc)))
 			{
-				return '<img class="color-scheme--dark" src="' . self::getUrl($darkSrc) . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="' . StringUtil::specialchars($alt) . '" loading="lazy"' . ($attributes ? ' ' . $attributes : '') . ' style="display:none"><img class="color-scheme--light" src="' . self::getUrl($src) . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="' . StringUtil::specialchars($alt) . '" loading="lazy"' . ($attributes ? ' ' . $attributes : '') . '>';
+				return '<img class="color-scheme--dark" src="' . self::getUrl($darkSrc) . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="' . StringUtil::specialchars($alt) . '" loading="lazy"' . ($attributes ? ' ' . $attributes : '') . '><img class="color-scheme--light" src="' . self::getUrl($src) . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="' . StringUtil::specialchars($alt) . '" loading="lazy"' . ($attributes ? ' ' . $attributes : '') . '>';
```

Comment:
```suggestion
				return '<img class="color-scheme--dark" src="' . self::getUrl($darkSrc) . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="' . StringUtil::specialchars($alt) . '" loading="lazy" style="display:none"' . ($attributes ? ' ' . $attributes : '') . '><img class="color-scheme--light" src="' . self::getUrl($src) . '" width="' . $objFile->width . '" height="' . $objFile->height . '" alt="' . StringUtil::specialchars($alt) . '" loading="lazy"' . ($attributes ? ' ' . $attributes : '') . '>';
```

Changing the order would also fix the issue, right?